images/image-repo-list-2004

```diff
@@ -3,8 +3,5 @@ gcAuthenticatedRegistry: e2eprivate
 etcdRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
```

Comment:
we can also remove`etcdRegistry` and `e2eRegistry` (maybe `sampleRegistry` too?. I don't think they are being used by Windows jobs anymore

Reply:
I suggest we do that in a separate PR. In case we break something, it will be easier to revert.